src/openqaoa-core/backends/plugin_finder.py

```diff
@@ -0,0 +1,36 @@
+def backend_boolean_list() -> list:
+    
+    try:
```

Comment:
Can we do all this programmatically?

basically, looking with modules of the form `openqaoa_*` and then creating all the variables programmatically?

Reply:
Yes this is possible. I also think its possible to go with a more robust/verbose solution that uses a dictionary with key, values here.